src/redux/reducers/initialState.js

```diff
@@ -63,6 +63,11 @@ const initialState = {
     updated: false,
     success: false
   },
+  followUnfollow: {
+    follow: false,
+    message: {},
+    error: {}
+  },
   articleRating: {
     response: null,
```

Comment:
What is `null`?

Reply:
@kimotho-njoki I am not sure, I will confirm with @PAbishai he was the one dealing with article Rating

Reply:
Wasn't this a story assigned to you and @pngg, why does @PAbishai have the answers?